service/src/main/java/com/myhome/controllers/CommunityController.java

```diff
@@ -194,6 +195,15 @@
       @PathVariable String communityId, @Valid @RequestBody
       AddCommunityAdminRequest request) {
     log.trace("Received request to add admin to community with id[{}]", communityId);
+    String userId = (String) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
```

Comment:
@WOLFI3654 The idiomatic way of doing this in Spring is to add an AuthorizationFilter to the chain. 
Can you make the changes to add an AuthorizationFilter instead of adding the logic here?

Please refer to jmprathab/MyHome/pull/165 for reference implementation if required.